### PR TITLE
add document about json functions.

### PR DESCRIPTION
**Pull request by hicqu**

### CODE REVIEW

**Comment by shenli** [comment]
@hicqu Please resolve the conflicts.

File: sql/json-functions.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: TiDB 用户文档
+category: user guide
+---
+
+# JSON 相关的函数和语法糖
+
+| 函数名或语法糖               | 示例                           | 功能描述                                               |
+| ---------------------------- | ------------------------------ | ------------------------------------                   |
+| [JSON_EXTRACT][json_extract] | JSON_EXTRACT(doc, path)        | 从JSON文档中解出某一路径对应的子文档                   |
```
Comment (queenypingcap) [inline sql/json-functions.md]: Please add space between English and Chinese characters.

**Comment by zimulala** [comment]
@hicqu Please resolve the conflicts.

**Comment by hicqu** [comment]
@queenypingcap , PTAL thanks.

**Comment by shenli** [comment]
@hicqu Please resolve the conflicts.

**Comment by shenli** [comment]
@sunhao2017 PTAL

**Pull request merged**